src/aktualizr_lite/CMakeLists.txt

```diff
@@ -18,5 +19,7 @@ add_test(test_aktualizr-lite
 
 endif(BUILD_OSTREE)
 
-aktualizr_source_file_checks(main.cc)
+add_aktualizr_test(NAME lite-version SOURCES version_test.cc)
+
+aktualizr_source_file_checks(${SOURCES} version_test.cc)
```

Comment:
`main.cc` is missing here and it causes a configure error when `BUILD_OSTREE` is not defined